src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/xlf/PackageReference.xaml.cs.xlf

```diff
@@ -82,6 +82,16 @@
         <target state="translated">Umístění odkazovaného balíčku</target>
         <note />
       </trans-unit>
+      <trans-unit id="BoolProperty|GeneratePathProperty|DisplayName">
```

Comment:
Tagging @tmeschter.  Note that we don't currently have any more loc handbacks planned for 15.9, so this string would remain untranslated until Dev16.

Reply:
If the display name or description are going to be visible to users then we can't take this change. Loc cut-off was over a week ago.